File: Data Structures & Algorithms/sliding-window-maximum/submission-0.py
class Solution:
    def maxSlidingWindow(self, nums: List[int], k: int) -> List[int]:
        result= []
        start=0
        end=k-1

        while end<len(nums):
            current_max = max(nums[start:end+1])
            result.append(current_max)
            start+=1
            end+=1
        
        return result

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int l = 1;
        for(int r=l;r<nums.length;r++)   {
            if(nums[r]!=nums[r-1]){
                nums[l++]=nums[r];
            }
        }
        return l;
    }
}